docs/getting_started/setup.md

```diff
@@ -8,6 +8,7 @@ keywords:
   - monorepo
   - stacks
   - ensono
+  - cypress
```

Comment:
Should this be here? I don't think it was removed in the original PR
https://github.com/Ensono/amido.github.io/pull/552/files#diff-f14c9652c069db4d1a5dfc1566186930a26e68017ecaf9b0ba3cb46cb3c504be

Reply:
its just for searching, I put back in just as reinstated the cypress row in the table.